test/interpreter/function-varp.spec.ts

```diff
@@ -0,0 +1,26 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VAR.P', () => {
+  it('should take at least one argument', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.P()'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate variance (population)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.P(2, 3)'],
+      ['=VAR.P(2, 3, 4, TRUE(), FALSE(), "1",)'],
```

Comment:
`['=VAR.P(1)']` does not say anything about the condition. `0` is expected when `1` is an argument? 
```
it('should return zero if only one number is provided', () => {
it('should ignore empty cells, logical values, text or error values if they're a part of reference', () => {

```

Reply:
Zero from 1 arg is not any special case, its just part of how variance works. The only valid description is something like 'accepts one argument'.

Reply:
👍